callback_plugins/jsnapy.py

```diff
@@ -20,6 +20,9 @@ class CallbackModule(CallbackBase):
   CALLBACK_TYPE = 'aggregate'
   CALLBACK_NAME = 'jsnapy'
 
+  # only needed if you ship it and don't want to enable by default
```

Comment:
In what context "only needed if you ship"? we are shipping callback so its not optional.

Reply:
modified the comment.